AbstractionDemo.java

```diff
@@ -0,0 +1,35 @@
+package com.basicsstrong.oop;
+
+interface Mobile
+{
+	public void call(int number);
```

Comment:
What is the default access modifier for the methods inside the interface ?

Reply:
public ,static,final

Reply:
If the default is public, then why did you explicitly specified it as public ?

Reply:
yeah if not mentioned also not a problem, I didnot much notice on it